decryption-tool/decrypt.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+# To install the dependencies
+pip3 install -r ./src/requirements.txt
+
+# Sample command to run the script
+#sh decrypt.sh
+# shellcheck disable=SC2039
+# shellcheck disable=SC2162
+read -p 'Decryption level 3 or 4: ' level
+# shellcheck disable=SC2039
+if [ "$level" == 3 ]
```

Comment:
As suggested above remove the level understanding from the decrypt tool. Make it simple with either taking in an encryption key or pem file